htdocs/core/class/html.form.class.php

```diff
@@ -8369,6 +8369,7 @@ public function showLinkedObjectBlock($object, $morehtmlright = '', $compatibleI
 		global $conf, $langs, $hookmanager;
 		global $bc, $action;
 
+		$conf->project = (object) array("enabled"=>true);
```

Comment:
The project is an object managed differently in Dolibarr. It is linked to other object with the field fk_project. So using "wild" links like suggested by the feature to link object each other will create a duplicate way to make project link making confusion and unconsistency into data. 
Please introduce a hidden constant for this because this will lead to very dangerous situation.